### ISSUE TITLE
Syntax check not allowed with example phpunit.xml file

### ISSUE THREAD

**Issue by zoomyboy** [octobercms/docs#453]
When I copy the example phpunit.xml to a plugin and run my tests, I'm getting this warning:

```
PHPUnit 7.5.17 by Sebastian Bergmann and contributors.

  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 12:
  - Element 'phpunit', attribute 'syntaxCheck': The attribute 'syntaxCheck' is not allowed.

  Test results may not be as expected.
```

According to this post, the SyntaxCheck setting is actually deprecated: https://stackoverflow.com/questions/44328114/phpunit-what-does-syntaxcheck-configuration-parameter-stands-for-exactly

Should this be removed from the example file since PHPUnit doesnt seem to parse this at all?

**Comment by LukeTowers** [comment]
@bennothommo do the updated L6 docs include this change?

**Comment by zoomyboy** [comment]
The 6x branch kicked out a lot of phpunit settings - so the options are really limited in 6x.
However I cannot find any version which includes this - even if I go down to 5.6:
 
It's not in the L6 branch though:
https://github.com/laravel/laravel/blob/6.x/phpunit.xml
https://github.com/laravel/laravel/blob/5.6/phpunit.xml

**Comment by bennothommo** [comment]
@LukeTowers I'm pretty certain the L6 docs stipulate that line to be removed.

**Comment by bennothommo** [comment]
Fixed by #454.
